### PR TITLE
Appoint corporate secretary

### PR DESCRIPTION
**Pull request by rsuller**

### CODE REVIEW

File: cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+describe('Appoint a Corporate Secretary', () => {
+    it('File successful AP04', () => {
+        // Go to change registered office address
+        const companyOverview = new CompanyOverviewPage();
+        const appointCorporateSecretaryPage = new AppointCorporateSecretaryPage();
+
+        cy.accessibilityCheck();
+        companyOverview.selectAllForms().selectLinkWithText('Directors and secretaries')
+            .selectLinkWithText('Appointment of corporate secretary - AP04');
```
Comment (tlawrencech) [inline cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js]: I'm getting an error here that it cant find this link.  I think this is the one you mentioned to James and Ollie that the page actually displays Appointment of corporate secretary- AP04

Comment (rsuller) [reply cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js]: Yeah it is a defect it's the space around `-` modify it and rerun to see what the test does to see if content is correct or needs amending.

Comment (tlawrencech) [reply cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js]: Yes that worked, I pushed the changed to the link text

Comment (rsuller) [reply cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js]: I've made James and Ollie aware of this. It is inconsistent and therefore is a defect so should be fixed on Webfiling and not addressed in our test.

**Pull request merged**